src/ObjectUtils/PdbFileLlvm.h

```diff
@@ -46,7 +48,16 @@ class PdbFileLlvm : public PdbFile {
     return result;
   }
 
-  [[nodiscard]] uint32_t GetAge() const override { return session_->getGlobalScope()->getAge(); }
+  [[nodiscard]] uint32_t GetAge() const override {
```

Comment:
I think we should move the implementation of both `GetGuid` and `GetAge` to the .cpp file as it's now several lines.

Reply:
Done.